lib/ramble/ramble/application.py

```diff
@@ -519,6 +517,8 @@ def add_expand_vars(self, workspace):
                 if var not in self.variables.keys():
                     self.variables[var] = conf['default']
 
+    def _inject_chained_commands(self, executables):
```

Comment:
I think this should be called `_inject_commands`. It just builds out all of the commands for the experiment, not just chained commands (though it injects those too).

Reply:
Done